src/simtools/utils/general.py

```diff
@@ -455,6 +456,34 @@ def find_file(name, loc):
     raise FileNotFoundError(msg)
 
 
+def generate_list_of_files(file_names):
```

Comment:
Is "generate" the correct name? This doesn't generate a list of files but instead finds the files which are included in the wildcards, no?

Reply:
It generates a list of files using the wildcards. To emphasize the wildcard issue, I've renamed it to resolve_file_patterns()

Reply:
Much better, thanks!